recipes/ofxparse/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ofxparse" %}
+{% set version = "0.20" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: eb45db5802938ac0ab3664418ca564619cc9e7ec6d301c1063ffc16e587ec37e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3
```

Comment:
It appears that this library supports `python >=2.7`. For the greatest flexibility of users who might still be running python 2, it's probably best if you change it here and under `run` below unless there is a specific reason to make it `python >=3`.

Reply:
Indeed! Thanks for the comment, updated!